src/app/visual/editable-grid/editable-grid.component.html

```diff
@@ -1,37 +1,39 @@
 <div
   class="editable-grid"
 >
-  <sky-toolbar>
-    <sky-toolbar-item *ngIf="!editMode">
-      <button
-        class="sky-btn sky-btn-primary"
-        id="edit-btn"
-        (click)="setEditMode(true)"
-      >
-        Edit
-      </button>
-    </sky-toolbar-item>
-    <sky-toolbar-item *ngIf="editMode">
-      <button
-        class="sky-btn sky-btn-primary"
-        (click)="saveData();"
-      >
-        Save
-      </button>
-    </sky-toolbar-item>
-    <sky-toolbar-item *ngIf="editMode">
-      <button
-        class="sky-btn sky-btn-default"
-        (click)="cancelEdits();"
-      >
-        Cancel
-      </button>
-    </sky-toolbar-item>
-  </sky-toolbar>
-  <ag-grid-angular
-    [ngClass]="{'sky-ag-grid-editable': editMode}"
-    [gridOptions]="gridOptions"
-    [rowData]="gridData"
-  >
-  </ag-grid-angular>
+  <sky-ag-grid-wrapper>
+    <sky-toolbar>
+      <sky-toolbar-item *ngIf="!editMode">
+        <button
```

Comment:
Now that this button is inside the wrapper, I can't navigate to it with the `tab` key.

Reply:
good catch. I initially was trying to get the toolbar to be sticky in this PR, but after discussing with Paul I think that functionality belongs in data manager. This way the sky ag grid wrapper can focus on _just_ the grid.